### ISSUE TITLE
I updated IntelliJ + Cursive and can't run REPL

### ISSUE THREAD

**Issue by romariolopezc** [cursive-ide/cursive#2231]
#### IntelliJ
```
IntelliJ IDEA 2019.2.1 (Community Edition)
Build #IC-192.6262.58, built on August 20, 2019
Runtime version: 11.0.3+12-b304.39 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Registry: ide.mac.allowDarkWindowDecorations=true
Non-Bundled Plugins: IdeaVIM, com.arcticicestudio.nord.jetbrains, com.markskelton.one-dark-theme, com.wakatime.intellij.plugin, org.intellij.scala, com.cursiveclojure.cursive
```

#### Cursive
```
Built on: 2019-07-25 12:31
Built from: 1.8.2
```

#### Error
```
Syntax error compiling at (mranderson048/toolsreader/v1v1v1/clojure/tools/reader/default_data_readers.clj:1:1).
	at clojure.lang.Compiler.load(Compiler.java:7647)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:428)
	at clojure.core$load$fn__6824.invoke(core.clj:6126)
	at clojure.core$load.invokeStatic(core.clj:6125)
	at clojure.core$load.doInvoke(core.clj:6109)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5908)
	at clojure.core$load_one.invoke(core.clj:5903)
	at clojure.core$load_lib$fn__6765.invoke(core.clj:5948)
	at clojure.core$load_lib.invokeStatic(core.clj:5947)
	at clojure.core$load_lib.doInvoke(core.clj:5928)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$load_libs.invokeStatic(core.clj:5985)
	at clojure.core$load_libs.doInvoke(core.clj:5969)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$require.invokeStatic(core.clj:6007)
	at clojure.core$require.doInvoke(core.clj:6007)
	at clojure.lang.RestFn.invoke(RestFn.java:482)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader$eval1860$loading__6706__auto____1861.invoke(reader.clj:9)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader$eval1860.invokeStatic(reader.clj:9)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader$eval1860.invoke(reader.clj:9)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7165)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:428)
	at clojure.core$load$fn__6824.invoke(core.clj:6126)
	at clojure.core$load.invokeStatic(core.clj:6125)
	at clojure.core$load.doInvoke(core.clj:6109)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5908)
	at clojure.core$load_one.invoke(core.clj:5903)
	at clojure.core$load_lib$fn__6765.invoke(core.clj:5948)
	at clojure.core$load_lib.invokeStatic(core.clj:5947)
	at clojure.core$load_lib.doInvoke(core.clj:5928)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$load_libs.invokeStatic(core.clj:5985)
	at clojure.core$load_libs.doInvoke(core.clj:5969)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$require.invokeStatic(core.clj:6007)
	at clojure.core$require.doInvoke(core.clj:6007)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at mranderson048.toolsnamespace.v0v3v0_alpha3.clojure.tools.namespace.parse$eval1852$loading__6706__auto____1853.invoke(parse.cljc:9)
	at mranderson048.toolsnamespace.v0v3v0_alpha3.clojure.tools.namespace.parse$eval1852.invokeStatic(parse.cljc:9)
	at mranderson048.toolsnamespace.v0v3v0_alpha3.clojure.tools.namespace.parse$eval1852.invoke(parse.cljc:9)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7165)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:428)
	at clojure.core$load$fn__6824.invoke(core.clj:6126)
	at clojure.core$load.invokeStatic(core.clj:6125)
	at clojure.core$load.doInvoke(core.clj:6109)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5908)
	at clojure.core$load_one.invoke(core.clj:5903)
	at clojure.core$load_lib$fn__6765.invoke(core.clj:5948)
	at clojure.core$load_lib.invokeStatic(core.clj:5947)
	at clojure.core$load_lib.doInvoke(core.clj:5928)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$load_libs.invokeStatic(core.clj:5985)
	at clojure.core$load_libs.doInvoke(core.clj:5969)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$require.invokeStatic(core.clj:6007)
	at clojure.core$require.doInvoke(core.clj:6007)
	at clojure.lang.RestFn.invoke(RestFn.java:703)
	at refactor_nrepl.core$eval1844$loading__6706__auto____1845.invoke(core.clj:1)
	at refactor_nrepl.core$eval1844.invokeStatic(core.clj:1)
	at refactor_nrepl.core$eval1844.invoke(core.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7165)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:428)
	at clojure.core$load$fn__6824.invoke(core.clj:6126)
	at clojure.core$load.invokeStatic(core.clj:6125)
	at clojure.core$load.doInvoke(core.clj:6109)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5908)
	at clojure.core$load_one.invoke(core.clj:5903)
	at clojure.core$load_lib$fn__6765.invoke(core.clj:5948)
	at clojure.core$load_lib.invokeStatic(core.clj:5947)
	at clojure.core$load_lib.doInvoke(core.clj:5928)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$load_libs.invokeStatic(core.clj:5985)
	at clojure.core$load_libs.doInvoke(core.clj:5969)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$require.invokeStatic(core.clj:6007)
	at clojure.core$require.doInvoke(core.clj:6007)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at refactor_nrepl.plugin$eval1838$loading__6706__auto____1839.invoke(plugin.clj:1)
	at refactor_nrepl.plugin$eval1838.invokeStatic(plugin.clj:1)
	at refactor_nrepl.plugin$eval1838.invoke(plugin.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7165)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:428)
	at clojure.core$load$fn__6824.invoke(core.clj:6126)
	at clojure.core$load.invokeStatic(core.clj:6125)
	at clojure.core$load.doInvoke(core.clj:6109)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5908)
	at clojure.core$load_one.invoke(core.clj:5903)
	at clojure.core$load_lib$fn__6765.invoke(core.clj:5948)
	at clojure.core$load_lib.invokeStatic(core.clj:5947)
	at clojure.core$load_lib.doInvoke(core.clj:5928)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$load_libs.invokeStatic(core.clj:5985)
	at clojure.core$load_libs.doInvoke(core.clj:5969)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$require.invokeStatic(core.clj:6007)
	at clojure.core$require.doInvoke(core.clj:6007)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at leiningen.core.utils$require_resolve.invokeStatic(utils.clj:102)
	at leiningen.core.utils$require_resolve.invoke(utils.clj:95)
	at leiningen.core.project$apply_middleware.invokeStatic(project.clj:816)
	at leiningen.core.project$apply_middleware.invoke(project.clj:810)
	at clojure.core.protocols$naive_seq_reduce.invokeStatic(protocols.clj:62)
	at clojure.core.protocols$interface_or_naive_reduce.invokeStatic(protocols.clj:72)
	at clojure.core.protocols$fn__8144.invokeStatic(protocols.clj:169)
	at clojure.core.protocols$fn__8144.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__8099$G__8094__8108.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__8131.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__8131.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__8073$G__8068__8086.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6828)
	at clojure.core$reduce.invoke(core.clj:6810)
	at leiningen.core.project$apply_middleware.invokeStatic(project.clj:812)
	at leiningen.core.project$apply_middleware.invoke(project.clj:810)
	at leiningen.core.project$activate_middleware.invokeStatic(project.clj:844)
	at leiningen.core.project$activate_middleware.invoke(project.clj:840)
	at leiningen.core.project$set_profiles.invokeStatic(project.clj:929)
	at leiningen.core.project$set_profiles.doInvoke(project.clj:922)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at leiningen.core.project$merge_profiles.invokeStatic(project.clj:937)
	at leiningen.core.project$merge_profiles.invoke(project.clj:931)
	at leiningen.repl$repl.invokeStatic(repl.clj:377)
	at leiningen.repl$repl.doInvoke(repl.clj:316)
	at clojure.lang.RestFn.invoke(RestFn.java:425)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at leiningen.core.main$partial_task$fn__6592.doInvoke(main.clj:284)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at leiningen.core.main$apply_task.invokeStatic(main.clj:334)
	at leiningen.core.main$apply_task.invoke(main.clj:320)
	at leiningen.core.main$resolve_and_apply.invokeStatic(main.clj:343)
	at leiningen.core.main$resolve_and_apply.invoke(main.clj:336)
	at cursive.leiningen.task$task_details$fn__1661.invoke(task.clj:69)
	at cursive.leiningen.task$task_details.invokeStatic(task.clj:65)
	at cursive.leiningen.task$task_details.invoke(task.clj:35)
	at clojure.lang.Var.invoke(Var.java:388)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:124)
	at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:117)
	at cursive.shim.ShimBootstrap$Shim.apply(ShimBootstrap.java:53)
	at cursive.repl.runner.LocalConfiguration$createLeinParameters$ret$1.invoke(ReplRunConfigurations.kt:325)
	at cursive.repl.runner.LocalConfiguration$createLeinParameters$ret$1.invoke(ReplRunConfigurations.kt:261)
	at cursive.shim.ShimProvider.withShim(Shim.kt:99)
	at cursive.repl.runner.LocalConfiguration.createLeinParameters(ReplRunConfigurations.kt:324)
	at cursive.repl.runner.LocalConfiguration.access$createLeinParameters(ReplRunConfigurations.kt:261)
	at cursive.repl.runner.LocalConfiguration$getRunProfileState$1.createDeferredParameters(ReplRunConfigurations.kt:294)
	at cursive.runner.DeferredCommandLineState.cacheDeferredParameters(Deferred.kt:50)
	at cursive.runner.DeferredRunner$execute$1.run(Deferred.kt:86)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:894)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:169)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:156)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:408)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:294)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: java.sql.Timestamp
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at clojure.lang.RT.classForName(RT.java:2207)
	at clojure.lang.RT.classForNameNonLoading(RT.java:2220)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader.default_data_readers$eval2382$loading__6706__auto____2383.invoke(default_data_readers.clj:11)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader.default_data_readers$eval2382.invokeStatic(default_data_readers.clj:11)
	at mranderson048.toolsreader.v1v1v1.clojure.tools.reader.default_data_readers$eval2382.invoke(default_data_readers.clj:11)
	at clojure.lang.Compiler.eval(Compiler.java:7176)
	at clojure.lang.Compiler.eval(Compiler.java:7165)
	at clojure.lang.Compiler.load(Compiler.java:7635)
	... 204 more```

**Comment by scottlowe** [comment]
@romariolopezc There are various unresolved issues with **IntelliJ 2019.2.1**. See this issue:

https://github.com/cursive-ide/cursive/issues/2209

In the meantime, the best thing that you can do is to downgrade IntelliJ to the previous version that worked for you. I'm currently using **IntelliJ 2019.1.4**  + Cursive v1.8.2-2019.1

**Comment by cursive-ide** [comment]
Thanks @scottlowe, you're correct that this is a JDK version issue, but in this case it's the JDK used to run IntelliJ itself. The canonical place discussing all the 2019.x issues is [this thread](https://groups.google.com/d/topic/cursive/ntvsiLAehY0/discussion), I talk about this problem in the last paragraph there.

@romariolopezc It looks like you probably have some CIDER stuff kicking around, perhaps in a user profile? If you either upgrade or remove that (Cursive doesn't need it) then hopefully this should work.

**Comment by romariolopezc** [comment]
I fixed it by selection the `Boot JDK` to 1.8 😃
